sechub-api-java/build.gradle

```diff
@@ -90,8 +78,8 @@ task callOpenAPIJavaGenerator(type: org.openapitools.generator.gradle.plugin.tas
             licenseUrl					: "https://github.com/mercedes-benz/sechub/blob/develop/LICENSE",
             developerEmail				: "",
             developerName				: "",
-            developerOrganization 		: "Daimler TSS",
-            developerOrganizationUrl	: "https://www.daimler-tss.com/",
+            developerOrganization 		: "Mercedes-Benz Tech Innovation",
```

Comment:
Could you please fix the indention and replace tabs by spaces ? 
You could also setup eclipse to use always spaces instead of tabs, so this will  not happen again.